_studio/shared/src/libmfx_core.cpp

```diff
@@ -1381,7 +1381,45 @@ mfxStatus CoreDoSWFastCopy(mfxFrameSurface1 *pDst, mfxFrameSurface1 *pSrc, int c
 
         break;
 
+#if (MFX_VERSION >= 1027)
+    case MFX_FOURCC_Y210:
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
```

Comment:
And the whole block presents already below, line ~1470: https://github.com/Intel-Media-SDK/MediaSDK/pull/1788/commits/4ac1843e24f90d84a1e25559efafa3bf42784f84#diff-fae1222e835b7a0eea6af66261c4d4d0R1470